packages/components/pages/f-loyalty/src/store/loyalty.module.js

```diff
@@ -10,10 +10,33 @@ import {
 /**
  * @function resolveGlobalUserID
  * @param  {string} authToken - auth token.
+ * @param logger
+ * @param errorHandler
  */
-const resolveGlobalUserID = authToken => {
-    const tokenData = jwtDecode(authToken);
-    return tokenData?.global_user_id;
+const resolveGlobalUserID = (authToken, logger, errorHandler) => {
+    // eslint-disable-next-line no-useless-catch
```

Comment:
Is this comment still required, now we're performing an action and not rethrowing?

Reply:
good point